consensus/consensus_v2.go

```diff
@@ -18,24 +18,22 @@ import (
 	"github.com/pkg/errors"
 )
 
-// handlemessageupdate will update the consensus state according to received message
-func (consensus *Consensus) handleMessageUpdate(payload []byte) {
-	if len(payload) == 0 {
-		return
-	}
-	msg := &msg_pb.Message{}
-	if err := protobuf.Unmarshal(payload, msg); err != nil {
-		consensus.getLogger().Error().Err(err).Msg("Failed to unmarshal message payload.")
-		return
-	}
+var (
+	errInVCNotLeader    = errors.New("something vc not leader")
```

Comment:
These errors messages are just temp placeholders. Please update with more accurate message.

Reply:
this one actually shouldn't be an error, return nil